rel/value_tuple.go

```diff
@@ -157,30 +166,26 @@ var LexerNamePat = `([$@A-Za-z_][0-9$@A-Za-z_]*)`
 
 var identRE = regexp.MustCompile(`\A` + LexerNamePat + `\z`)
 
+func TupleNameRepr(name string) string {
+	if identRE.Match([]byte(name)) {
+		return name
+	}
+	data, err := json.Marshal(name)
```

Comment:
Is the purpose of `json.Marshal` to put quotes around it and escape some special characters?

Reply:
Existing code that was refactored. The better approach is to use repr, which I've just done.